### ISSUE TITLE
"Offline" store a specific folder

### ISSUE THREAD

**Issue by ValerioNeriGit** [astrada/google-drive-ocamlfuse#626]
Hi, 

I was wondering I there is a functionality to permanently keep a folder and its content stored and synced on the device (aside of the cache mechanism) in order to have quicker access to files I know I need quite often when needed.

Thank you

**Comment by astrada** [comment]
Nope, sorry. You should probably use a sync client for that feature (rclone, drive,...).
